src/infrastructure/channels/reliable_channel.rs

```diff
@@ -35,7 +35,7 @@ pub struct ReliableChannel {
     // congestion control
     rtt_measurer: RttMeasurer,
     congestion_data: SequenceBuffer<CongestionData>,
-    quality: NetworkQuality,
+    _quality: NetworkQuality,
```

Comment:
Do we have an issue open for this, if so can we add a TODO here?

Reply:
#96